apps/dashboard/src/lib/utils/services/courses/index.ts

```diff
@@ -155,6 +155,35 @@ export async function updateCourse(
   return course.logo;
 }
 
+export async function updateCollection(
```

Comment:
Create a services folder for `pathways` and don't add pathway related functionality in the course service files